S/SDPA/build_tarballs.jl

```diff
@@ -71,6 +71,12 @@ elif [[ ${target} == *linux* ]]; then
     export LDFLAGS="-ldl -lrt"
 fi
 
+# work around missing strtoll strtoull, see https://github.com/JuliaLang/julia/issues/48081
+if [[ "${target}" == *mingw* ]]; then
+    make -C deps install-csl
```

Comment:
```suggestion
```
You aren't actually needing to compile CSL here, just doing the library linkage should be enough I believe.